deploy.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+
+export KRUIZE_UI_IMAGE="docker.io/bharathappali/kruize-ui:central-config"
+KRUIZE_UI_MANIFEST_FILE="kruize-ui-nginx-setup.yml"
```

Comment:
Is this needed?

Reply:
yes we use envsubst to create the yaml and this is the file we update in dynamic and static folders